packages/react/kibana_context/theme/theme_provider.tsx

```diff
@@ -9,11 +9,13 @@
 import React, { useMemo } from 'react';
 import useObservable from 'react-use/lib/useObservable';
 
-import {
-  CurrentEuiBreakpointProvider,
-  EuiThemeProvider,
-  EuiThemeProviderProps,
-} from '@elastic/eui';
+import { EuiThemeProvider, EuiThemeProviderProps } from '@elastic/eui';
+// @ts-ignore EUI exports this component internally, but Kibana isn't picking it up its types
+import { useIsNestedEuiProvider } from '@elastic/eui/lib/components/provider/nested';
+// @ts-ignore EUI exports this component internally, but Kibana isn't picking it up its types
+import { emitEuiProviderWarning } from '@elastic/eui/lib/services/theme/warning';
+
+import { KibanaEuiProvider } from '@kbn/react-kibana-context-root/eui_provider';
```

Comment:
I had not intended for this to be consumed outside the package... but perhaps we should, with plenty of documentation about why one shouldn't be using it...?

Reply:
EUI's general philosophy is that our top-level exports are public first-class citizen APIs, and devs _can_ reach into internal sub-directories as needed, but it's kind of a "you break it you buy it" mentality.

In this case since this workaround is (hopefully) temporary and we're the ones reaching for our own component, it's hopefully not so bad? :) But more documentation is never a bad thing!